fish_prompt.fish

```diff
@@ -427,5 +445,6 @@ function fish_prompt -d 'bobthefish, a fish theme optimized for awesome'
   else
     __bobthefish_prompt_dir
   end
+  __bobthefish_prompt_vi
   __bobthefish_finish_segments
```

Comment:
it seems like it would be more consistent with the way powerline works, for example, to move the vi segment all the way to the left. that would also separate it from the git segment so you wouldn't have back-to-back green segments when in insert mode inside an up to date git repo.

that said, i don't use vi mode, so i don't have a strong sense of where would make the most sense to put it.